include/ibamr/INSVCStaggeredConservativeHierarchyIntegrator.h

```diff
@@ -169,6 +172,20 @@ class INSVCStaggeredConservativeHierarchyIntegrator : public INSVCStaggeredHiera
      */
     void registerMassDensitySourceTerm(SAMRAI::tbox::Pointer<IBTK::CartGridFunction> S_fcn);
 
+    /*!
+     * Returns the number of cycles to perform for the present time step.
+     */
+    int getNumberOfCycles() const;
+
+    /*!
+     * Get the convective operator being used by the integrator class.
+     *
+     * \note The class employs INSVCStaggeredConservativeMassMomentum integrator
```

Comment:
Please make sure the class name (`INSVCStaggeredConservativeMassMomentumIntegrator`?) is spelled out completely to get linkage in Doxygen.

Reply:
🤫I will fix it...